mitodl/micromasters
# Removed feature flag code related to the final grade algorithm

#### What are the relevant tickets?
Closes #2890 

#### What's this PR do?
Removes all code related to the feature flagging of the final grade algorithm and the old version of final grade calculation

#### How should this be manually tested?
`alter_data` can be used here to set final grades for some course runs. Examples:

```bash
alter_data set_to_passed --username=YOUR_USERNAME --course-title='Digital Learning 100' --grade=80
alter_data set_to_failed --username=YOUR_USERNAME --course-run-key='course-v1:MITx+Digital+Learning+100+Jan_2017' --grade=50
```

You can check the dashboard to make sure it reflects the final grades accurately. Testing on both FA and non-FA programs would be best.
